config/CUSTOM-RULES.md

```diff
@@ -115,8 +115,6 @@ Custom rules can have more complicated behavior. They can also include other rul
 
 (There is no `Galar Pokedex` rule, `-Nonexistent` covers it in Gen 8.)
 
-`Little Cup` - allow only Pokémon that can evolve and aren't evolved
```

Comment:
No, this can stay. It's distinct from `Gen 8 Little Cup` which copies all the rules, not just the evo rule.